docs/guide/adapters/README.md

```diff
@@ -222,8 +222,8 @@ _(in order of first appearance)_
     * [Coordinator firmware](https://github.com/Koenkk/Z-Stack-firmware/raw/master/coordinator/Z-Stack_3.x.0/bin/CC1352P2_CC2652P_launchpad_coordinator_20220219.zip)  
     * [Router firmware](https://github.com/Koenkk/Z-Stack-firmware/raw/master/router/Z-Stack_3.x.0/bin/CC1352P2_CC2652P_launchpad_router_20220125.zip)  
     * Flashing instructions; see "Flashing CC1352/CC2652/CC2538 based adapters" below
-    * [Description](https://z2m.wiki/)  
-    * [Buy](https://z2m.wiki/)  
+    * [Description](https://z2m.store/)  
+    * [Buy](https://z2m.store/)  
```

Comment:
I feel the url is highly misleading, it looks like if it is the official Zigbee2MQTT site. Can you consider a different URL or clearly mention on the site that this is not the official Zigbee2MQTT site?

Reply:
can you help me change to https://shop68536829.taobao.com/

Reply:
Done